st2common/in-requirements.txt

```diff
@@ -3,6 +3,7 @@ apscheduler
 python-dateutil
 eventlet
 jinja2
+jmespath
```

Comment:
We already use `jsonpath-rw` dependency in other places.

I think it would be a good idea to also use that for your new filter, if possible, to avoid additional dependencies in the standard distribution.

I would be fine with this dependency if it was part of a custom filter which is not shipped with the default distribution.